pkg/cli/config/initialize/disks.go

```diff
@@ -0,0 +1,308 @@
+package initialize
+
+import "fmt"
+
+const (
+	crays3cache = "CRAYS3CACHE"
+	conrun      = "CONRUN"
+	conlib      = "CONLIB"
+	k8slet      = "K8SLET"
+	cephetc     = "CEPHETC"
+	cephvar     = "CEPHVAR"
+	contain     = "CONTAIN"
+	volumeGroup = "metalvg0"
+	raidArray   = "/dev/md/AUX"
+)
```

Comment:
We should really parameterize this since it's parameterized in Dracut, but no one uses that feature, so it's somewhat moot but definitely out of scope.

Can you add a TODO or a JIRA to replace these constants with flags? Minor priority, doesn't need to be for 1.6.

Reply:
https://jira-pro.it.hpe.com:8443/browse/MTL-2489